docs/ci-cd-environment/agent-migration-to-ubuntu2004.md

```diff
@@ -0,0 +1,42 @@
+---
+description: This guide specifies the steps to change Semaphore 2.0 agent OS from Ubuntu 18.04 to Ubuntu 20.04
+---
+
+# Migration from Ubuntu 18.04 image to Ubuntu 20.04 image 
+
+If you've used Semaphore Ubuntu 18.04 image, this guide will outline the
+key differences and provide you with a direction to migrate to the Ubuntu 20.04 image.
+
+### You don't have to migrate
+
+We know how much you rely on Semaphore to do your work and don't want to impose
+migration timeline. We do not depricate the Ubuntu 18.04 image.
+
+### Benefits of using the Ubuntu 20.04
+
+Ubuntu 20.04 LTS (Focal Fossa) is the latest lts release for the Ubuntu linux
+server operating system.(At the time of writing.) 
+It has enhanced resource usage and newer kernel, compared to Ubuntu 18.04.
+Will be supported by Canonical until April 2025.
+If you want to read more about Ubuntu 20.04 check out (Ubuntu Blog)[https://ubuntu.com/blog/ubuntu-server-20-04]
+
+### Ubuntu 20.04 image on Semaphore 2.0
+
+The Ubuntu 20.04 image on Semaphore 2.0 is the successor of the Ubuntu 18.04 image.
+Its preinstalled software stack is reduced to the most recent versions. For the exact list 
```

Comment:
Its preinstalled stack includes the most recent versions for many languages and frameworks. (I think that's what you meant, please confirm),